CONTRIBUTING.md

```diff
@@ -24,6 +24,12 @@ standardized:
   - For blog posts, place the file in `_posts/<language code>/`
 - Set the `lang` field to `<language code>`
 - Append `-<language code>` to both the `slug` and `name` fields
+- When linking internally, link to the appropriate translation, if one is available.
+  - When linking to a specific header/bullet within a page, #anchor links are [automatically generated](https://github.com/bitcoinops/bitcoinops.github.io/blob/master/_plugins/auto-anchor.rb).
+      - Due to [limitations](https://github.com/bitcoinops/bitcoinops.github.io/pull/349)
+        in the automatic anchoring process, some languages [need HTML comments
+        inserted](https://github.com/bitcoinops/bitcoinops.github.io/commit/4e450d1a1f72219ec50ad91edae605647164d25d#diff-435f99f277721eff9e2f244149575f41R41)
```

Comment:
I suggest "some languages need HTML comments inserted. See for example the `<!--1-->` [comment](https://github.com/bitcoinops/bitcoinops.github.io/commit/4e450d1a1f72219ec50ad91edae605647164d25d#diff-435f99f277721eff9e2f244149575f41R41) in this Japanese newsletter." to be absolutely clear about what you're demonstrating in the link.

Reply:
I think that’s clearer, thanks for the suggestion. Changed.